app/forms/serializers/forms.py

```diff
@@ -116,6 +117,10 @@ class Meta:
         model = EventForm
         fields = FormSerializer.Meta.fields + ("event", "type",)
 
+    def to_representation(self, instance):
```

Comment:
Tror dette er mulig istedenfor hentet fra registrations ![Screenshot_20210920-002859_GitHub.jpg](https://user-images.githubusercontent.com/60223374/133945041-1d0e650f-c9bd-43df-8c36-037f8c902792.jpg)

Reply:
Men vil det funke med at man også har input med event: number?

Reply:
Tror den skal klare det men test

Reply:
Får følgende `409 Conflict`:
![image](https://user-images.githubusercontent.com/31009729/134036819-1ffcf619-2f7c-4f21-872b-2cba03a48116.png)

Uansett hvordan jeg formaterer input:
![image](https://user-images.githubusercontent.com/31009729/134036612-73aac181-76d4-4c30-8bdb-21288c8f34a6.png)
![image](https://user-images.githubusercontent.com/31009729/134036656-e440c24f-553a-440b-81cf-60deb02e4b48.png)
![image](https://user-images.githubusercontent.com/31009729/134036691-6988b185-5806-464f-b040-9d3046de7682.png)

Når `EventListSerializer` ser ut som dette:
![image](https://user-images.githubusercontent.com/31009729/134036965-43c0991d-4a54-4261-bf0f-153f7538174e.png)

Noen idé om hvordan det evt kan funke? @Her0elt @Eirsteir

Jeg vil jo si at `to_representation` var ganske clean og funker veldig bra 🤷‍♂️

Reply:
Prøv å legge til source som i eksemple

Reply:
Source funket ikke
![image](https://user-images.githubusercontent.com/31009729/134041601-12903ead-9c6b-4162-8482-7b116690e5cf.png)

Reply:
Hvis du har read_only=True i en related serializer så vil ikke feltet bli sendt med til metoden som muterer modellen(FormSerializer.create()), derav 409 Conflict fordi event_id er null. 

Tror det beste her er å lage en ny serializer for GET eller gjøre som du har gjort her.

Reply:
Det gir mening. Er det mulig å lage en GET-only serializer med polymorphic?

Reply:
Noe sånn muligens, og så overstyre `get_serializer_class()` i ViewSet'et. 
```suggestion
class ReadOnlyFormPolymorphicSerializer(FormPolymorphicSerializer):
    resource_type = serializers.CharField()
    resource_type_field_name = "resource_type"
    model_serializer_mapping = {
        Form: FormSerializer,
        EventForm: EventFormListSerializer,
    }
 ```

Reply:
Bør det gjøres ift å bare legge til de 3 linjene med `to_representation` ? Føles mye mer tingvindt

Reply:
Det er kanskje greit når serializeren blir stor og veldig annerledes, men det går fint når det er en så liten endring. Ville gått for `to_representation` her